xml/Microsoft.VisualBasic/Strings.xml

```diff
@@ -1295,7 +1295,7 @@
 |`Text`|1|Performs a text comparison|
   
 ## Examples  
- This example uses the `InStr` function to return the position of the first occurrence of one string within another.  
+ This example uses the `InStr` function to return the position of the first occurrence of one string within another. In the first example, the search starts from the fourth character and returns the next lower case p because `CompareMethod` is `Text` and is therefore case insensitive. The position is always relative to the beginning of the string, regardless of the start position.
```

Comment:
```suggestion
 This example uses the `InStr` function to return the position of the first occurrence of one string within another. In the first example, the search starts from the fourth character and returns the next lower case "p" because `CompareMethod` is `Text` and therefore case insensitive. The position is always relative to the beginning of the string, regardless of the start position.
```